tests/functional.mk

```diff
@@ -62,4 +84,25 @@ BGB_ADDITIONAL_GO_ENV := CGO_ENABLED=0
 
 include makelib/build_go_bin.mk
 
+# 1 - image
+# 2 - aci directory
+# 3 - ace validator
+define FTST_GENERATE_ACE_IMAGE
+$1: ACTOOL := $$(ACTOOL)
```

Comment:
Maybe replace this line with `$$(call forward-vars,$1,ACTOOL)` (mind the double dollar sign).